### PR TITLE
feat: Add cart badge

### LINKED ISSUE #301: Add totalItemQuantity badge to cart icon in nav bar
## Work
Add totalItemQuantity badge to cart icon in nav bar. See Zeplin designs: https://app.zeplin.io/project/5afc82f5e8d73e250ff9d855/screen/5b36745aa148a74f305421c4

Use MUI badge? Data should be on `cart.totalItemQuantity`

### PR DESCRIPTION
**Pull request by rosshadden**
Resolves #301 
Impact: **minor**
Type: **feature**

## Issue
Cart icon has no quantity badge.

## Solution
Added a quantity badge.

## Breaking changes
None.

### CODE REVIEW

File: src/components/MiniCart/MiniCart.js [hunk 1]
```diff
@@ -146,9 +153,10 @@ export default class MiniCart extends Component {
   }
 
   render() {
-    const { classes, uiStore } = this.props;
+    const { cart, classes, uiStore } = this.props;
     const { isCartOpen } = uiStore;
-    const id = isCartOpen ? "simple-popper" : null;
+    const id = (isCartOpen) ? "simple-popper" : null;
+    const cartQuantity = (cart && Array.isArray(cart.items) && cart.items.length) || 0;
```
Comment (mikemurray) [inline src/components/MiniCart/MiniCart.js]: Use `cart.totalItemQuantity`

Comment (rosshadden) [reply src/components/MiniCart/MiniCart.js]: That didn't exist. I looked for it but couldn't find it.

Comment (rosshadden) [reply src/components/MiniCart/MiniCart.js]: Hmm that's weird. It definitely wasn't there when I checked before. I had tried both logging and also a debugger. But lo and behold, it's there now.

**Pull request merged**